### PR TITLE
Initial release of CVE-2019-18180

### PR DESCRIPTION
**Pull request by jensoliver**

### CODE REVIEW

**Comment by cve-team** [comment]
jensoliver is not recognized currently as an account participating in the Git Pilot. To ensure timely processing of your pull request, please contact the CVE Team using either the CVE Request web form (https://cveform.mitre.org/) or an email (cve@mitre.org) and confirm your identity.

**Comment by jensoliver** [comment]
I sent an email and also used the form to verify my account for CNA usage

File: 2019/18xxx/CVE-2019-18180.json [hunk 1]
```diff
@@ -0,0 +1,116 @@
+{
```
Comment (cve-team) [inline 2019/18xxx/CVE-2019-18180.json]: CNA(s) of user (OTRS) do not match owning CNA.

**Comment by jensoliver** [comment]
solved

**Pull request closed without merging**